community.json

```diff
@@ -1198,6 +1198,13 @@
         "state": "working",
         "url": "https://github.com/Kloadut/shout_ynh"
     },
+    "shsd": {
+        "branch": "master",
+        "level": 1,
```

Comment:
Thanks !

I just think that you should not add the `"level"` key yourself as it will actually be computed by the automatic tests running on https://ci-apps.yunohost.org/jenkins/ (c.f. also https://dash.yunohost.org/appci/branch/stable ) :wink: